src/Microsoft.Data.SqlClient/tests/ManualTests/SQL/InstanceNameTest/InstanceNameTest.cs

```diff
@@ -87,6 +87,7 @@ public static void ConnectManagedWithInstanceNameTest(bool useMultiSubnetFailove
         }
 
 #if NETCOREAPP
+        [ActiveIssue("27824")] // When specifying instance name and port number, this method call always returns false
```

Comment:
Does that mean issue https://github.com/dotnet/SqlClient/issues/2187 should be re-opened too?

Reply:
PR #2240 was the one that addressed #2187 and I am not sure why the code change was needed when the test was fixed.